aws/ci_build.rake

```diff
@@ -234,7 +234,16 @@ end
 task :chef_update do
   if CDO.daemon && CDO.chef_managed
     RakeUtils.with_bundle_dir(cookbooks_dir) do
-      RakeUtils.bundle_exec 'berks', 'install'
```

Comment:
ah, good catch, it probably still does! I'll need to move this line outside of the staging-only block (or create two blocks surrounding it).